src/plugins/hetero/executable_network.cpp

```diff
@@ -73,6 +88,8 @@ HeteroExecutableNetwork::HeteroExecutableNetwork(const InferenceEngine::CNNNetwo
     // hetero_config, device_config and user_config are unchanged global and local configs set by user
     // we need to create _hetero_config and _device_config based on them, which will
     // contain only hetero (_hetero_config) and only device (_device_config) properties
+    auto _heteroPlugin = std::make_shared<HeteroPlugin::Engine>();
```

Comment:
Why?

Reply:
It was created in order to use `MergeConfigs` method. Fully replaced with new Plugin